packages/xrpl/test/models/permissionedDomainSet.test.ts

```diff
@@ -0,0 +1,94 @@
+import { stringToHex } from '@xrplf/isomorphic/dist/utils'
+import { assert } from 'chai'
+
+import { AuthorizeCredential, validate, ValidationError } from '../../src'
+
+/**
+ * PermissionedDomainSet Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('PermissionedDomainSet', function () {
+  let tx
+  const sampleCredential: AuthorizeCredential = {
+    Credential: {
+      CredentialType: stringToHex('Passport'),
+      Issuer: 'rfmDuhDyLGgx94qiwf3YF8BUV5j6KSvE8',
+    },
+  }
+
+  beforeEach(function () {
+    tx = {
+      TransactionType: 'PermissionedDomainSet',
+      Account: 'rfmDuhDyLGgx94qiwf3YF8BUV5j6KSvE8',
+      DomainID:
+        'D88930B33C2B6831660BFD006D91FF100011AD4E67CBB78B460AF0A215103737',
+      AcceptedCredentials: [sampleCredential],
+    } as any
+  })
+
+  it('verifies valid PermissionedDomainSet', function () {
+    assert.doesNotThrow(() => validate(tx))
+  })
+
+  it(`throws with invalid field DomainID`, function () {
+    // DomainID is expected to be a string
+    tx.DomainID = 1234
+    const errorMessage = 'PermissionedDomainSet: invalid field DomainID'
+    assert.throws(() => validate(tx), ValidationError, errorMessage)
+    assert.throws(() => validate(tx), ValidationError, errorMessage)
```

Comment:
This doesn't do anything.
```suggestion
    assert.throws(() => validatePermissionedDomainSet(tx), ValidationError, errorMessage)
    assert.throws(() => validate(tx), ValidationError, errorMessage)
```

Reply:
I have removed the second, duplicate function call. `validate` method invokes `validatePermissionedDomainSet` in its implementation. There is no need to invoke this method again.

[9154151](https://github.com/XRPLF/xrpl.js/pull/2874/commits/91541510cb43525583d0c32c3acec258831fd230)